towerlib/entities/job.py

```diff
@@ -1911,3 +1967,8 @@ def module_name(self):
 #              u'notifications': u'/api/v2/ad_hoc_commands/4979/notifications/',
 #              u'relaunch': u'/api/v2/ad_hoc_commands/4979/relaunch/',
 #              u'stdout': u'/api/v2/ad_hoc_commands/4979/stdout/'},
+
+@dataclass
```

Comment:
This should probably be in the main entities file because i have the feeling that more resources would have labels and they could reuse the same data definition.

Reply:
what is the main entities file? towerlib.py ?

Reply:
towerlib/entities/core.py
you also need to import it in the __init__.py file like the other resources. That was it will be accessible under the entities namespace so it can be used by other resources if needed without fear of cyclic references.

Reply:
Done
![image](https://user-images.githubusercontent.com/18288421/123946300-58eebb00-d99f-11eb-9356-b802a8f25810.png)